subsys/disk/disk_access_stm32_sdmmc.c

```diff
@@ -64,6 +64,8 @@ static int stm32_sdmmc_clock_enable(struct stm32_sdmmc_priv *priv)
 		;
 
 	LL_RCC_SetSDMMCClockSource(LL_RCC_SDMMC1_CLKSOURCE_PLLSAI1);
+#elif defined(CONFIG_SOC_SERIES_STM32H7X)
+	LL_RCC_SetSDMMCClockSource(LL_RCC_SDMMC_CLKSOURCE_PLL2R);
```

Comment:
PLL1Q or PLL2R can be selected, so it should probably be configurable? 

![image](https://user-images.githubusercontent.com/25011557/102665712-97502580-4185-11eb-998c-446d6160dfc0.png)

Reply:
Yes, there are many ways to configure this clock. Don't know whether this is good moment to introduce too many options, it is so complex it probably would make sense to introduce PLL child object under RCC and give clock source selection for each driver. Good time will be after SDMMC and QSPI work together.

Reply:
I added option to select clock source, might be already useful